samples/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,55 @@
+# Using Microsoft XmlSerializer Generator on .Net Core
+
+When a running application creates an instance of XmlSerializer, it generates a temporary assembly containing the implementation of the serializer. This happens once for each type for which an XmlSerializer is needed. To improve performance, we can use Microsoft XmlSerializer Generator to pre-generate code for an XmlSerializer for each specified type in a project and compile it into an assembly. This assembly will then be used by the application when creating an XmlSerializer instead of generating a temporary one.
```

Comment:
A sample app would be useful so that people could try out these instructions w/o needing to create anything special themselves. Kinda like these samples: https://github.com/dotnet/dotnet-docker-samples